src/test/K6/src/tests/platform/events/subscriptions.js

```diff
@@ -57,6 +57,8 @@ export default function (data) {
   });
   addErrorCount(success);
 
+  sleep(20);
+
```

Comment:
Was the issue that deleteSubscriptionById as called before the check call on line 54 was completed?

Reply:
the issue was that, the azure function for validating a subscription tried to update the subscription, but before that the test script has deleted the subscription using `deleteSubscriptionById`. the check in line 54 is a test on the response from `getsubscription`

Reply:
Ah, ok. Thx.